src/komponenter/oppholdstillatelse/kontakt-meg-melding.tsx

```diff
@@ -41,17 +43,21 @@ class KontaktMegOppholdstillatelse extends React.Component<AllProps> {
         const oppgaveStatus = this.props.oppgaveStatus.status
         const oppgaveStatusCode = this.props.oppgaveStatus.data.status
         const OppgaveSuccess = () => {
+          const idag = new Date()
+          const nesteVirkedag = virkedager(idag, 2)
```

Comment:
Trenger nesten ikke argument nr 2 siden det er default. Men du tenker kanskje at det er for leseligheten sin skyld?

Reply:
for leselighet, og spesielt siden det kan hende den spretter til 3 dager... da er det kjapt å se hvor man må fikse (uten å gå via README)